### DISCUSSION TITLE
gltf编辑器与查看器都正常显示，在Chrome上显示为黑色

### DISCUSSION THREAD

**Discussion by paomian7fenshu** [galacean/discussion#24] [Engine] [answered]
已经设置环境光
 engine.sceneManager.activeScene.ambientLight.diffuseSolidColor.setValue(1, 1, 1, 1);
 engine.sceneManager.activeScene.ambientLight.diffuseIntensity = 1;
 engine.sceneManager.activeScene.ambientLight.diffuseMode = DiffuseMode.SolidColor;

文件链接：
https://g.alicdn.com/eva-assets/f56b23d9750acb36c4b84c622db18026/0.0.1/qingyuan/Downloads/test/C13_low3(1).gltf

![image](https://user-images.githubusercontent.com/46734476/195754575-7d47eab1-be6a-4ddd-9a62-bcf686d1e9da.png)
![image](https://user-images.githubusercontent.com/46734476/195754709-7c101f28-bbd5-4e57-aee7-18582e94bb9e.png)

**Comment by zhuxudong** [comment, marked as answer]
* 需要加一个IBL,才能镜面反射，添加方法参考 https://oasisengine.cn/0.8/docs/ambient-light-cn
* 之所以不加的时候是黑色，是因为这个模型的材质金属度为1，金属材质 =  0 漫反射+ 1镜面反射
